android/src/main/java/com/baumblatt/capacitor/firebase/auth/CapacitorFirebaseAuth.java

```diff
@@ -44,16 +44,16 @@
 
     private boolean nativeAuth = false;
 
-    private Config config;
+    private CapConfig config;
 
-    public Config getConfig() {
+    public CapConfig getConfig() {
         return this.config;
     }
 
     public void load() {
         super.load();
 
-        this.config = new Config(this.bridge.getActivity().getAssets(), null);
+        this.config = new CapConfig(this.bridge.getActivity().getAssets(), null);
```

Comment:
Is this backwards compatible?  If not, then ideally this would be able to check the version and select which class to use.

Reply:
I like this idea in same cases, but in this one, I think for maintainability reasons it's best to not doing it. The old `Config` class has changed too. It's not the same as in the previous version, so making it backwards compatible can't guarantee it will work 100% exactly as expected and it will eventually removed from the project.

I'd like to add a note on the `Readme` file saying explicitly in which version the changes happened so anyone who uses the project can know which version to install (always talking about old versions).

Thoughts?

Reply:
Maybe it's not a huge deal - the change to use a Config instance the way it does only happened with version 2.2 anyway?  I guess that change just switch to use Config() instead of CapConfig(), so this could be seen as more of a completion of that change.  There's already a note about that broken upgrade path in the Readme.

Reply:
Yeah, we introduced a breaking change in 2.2.0 without noticing, and reverted it in 2.2.1. 
I would recommend switching to using Config class the way you used it before, it will only be a problem for 2.2.0 users.
Or if you use CapConfig, then require Capacitor 2.2.1 or newer.

Reply:
I am using the latest Capacitor(2.3.x) wiith 2.3.1 .. Android Studio complains (No CapConfig)

Reply:
> Yeah, we introduced a breaking change in 2.2.0 without noticing, and reverted it in 2.2.1.
> I would recommend switching to using Config class the way you used it before, it will only be a problem for 2.2.0 users.
> Or if you use CapConfig, then require Capacitor 2.2.1 or newer.

Doesn't work with 2.3.0